File: GotravelDesktop/src/main/java/com/gotravel/Model/Usuario.java
package com.gotravel.Model;

import lombok.*;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Usuario {

    private int id;
    private String nombre;
    private String apellidos;
    private String email;
    private String contrasena;
    private List<Rol> roles;
    private String tfno;
    private byte[] foto;
    private boolean profesional;
    private boolean administrador;
    private String oculto = "0";

    public Usuario(int id, String nombre, String apellidos, String email, String contrasena, String tfno) {
        this.id = id;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.email = email;
        this.tfno = tfno;
        this.contrasena = contrasena;
    }

    public String getApellidos() {
        return apellidos != null ? apellidos : "";
    }

    public String getTfno() {
        return tfno != null ? tfno : "";
    }
}

File: old/ClienteMovil/GoTravel/app/src/main/java/com/gotravel/clienteMovil/data/Usuario.kt
package com.gotravel.clienteMovil.data

import kotlinx.serialization.Serializable

@Serializable
class Usuario (

    val id: Int,
    val nombre: String,
    val apellidos: String?,
    val email: String,
    val tfno: String?,
    val foto: Array<Byte>?,
    val oculto: String

)